R/dfm-methods.R

```diff
@@ -87,6 +87,8 @@ as.dfm.default <- function(x) {
 #' @method as.dfm dfm
 #' @export
 as.dfm.dfm <- function(x) {
+    # make sure those dimensions are character
+    x@Dimnames <- lapply(x@Dimnames, as.character)
```

Comment:
Why do you use `lapply` instead of `set_dfm_dimnames()`?

Reply:
OK I changed it in 789b9ea but the RHS still needs to access the slot directly since `as.dfm()` is a very low-level function.